libnavigation-base/api/current.txt

```diff
@@ -269,6 +269,7 @@ package com.mapbox.navigation.base.metrics {
   public interface MetricEvent {
     method public String getMetricName();
     method public String toJson(com.google.gson.Gson gson);
+    method public com.mapbox.bindgen.Value toValue();
```

Comment:
I would hide this from the public API. any objections @dzinad @VysotskiVadim @LukasPaczos ?